starboard/nplb/nplb_evergreen_compat_tests/checks.h

```diff
@@ -19,9 +19,11 @@
 
 #if SB_IS(EVERGREEN_COMPATIBLE)
 
+#if SB_API_VERSION < 16
```

Comment:
Let's add the SB_CHECK to 
starboard/nplb/nplb_evergreen_compat_tests/executable_memory_test.cc

Reply:
Added in test fixture